airflow/dag_processing/bundles/local.py

```diff
@@ -19,21 +19,25 @@
 
 from pathlib import Path
 
+from airflow import settings
 from airflow.dag_processing.bundles.base import BaseDagBundle
 
 
 class LocalDagBundle(BaseDagBundle):
     """
     Local DAG bundle - exposes a local directory as a DAG bundle.
 
-    :param local_folder: Local folder where the DAGs are stored
+    :param directory: Local directory where the DAGs are stored
```

Comment:
path feels a little more familiar but 🤷

Reply:
Good call, I was taking inspiration from `os`, but I must not have looked very well because I see way more `path` now that I look again 🙃